plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaSecurityConfig.java

```diff
@@ -0,0 +1,151 @@
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+import static io.airlift.configuration.testing.ConfigAssertions.assertFullMapping;
+import static io.airlift.configuration.testing.ConfigAssertions.assertRecordedDefaults;
+import static io.airlift.configuration.testing.ConfigAssertions.recordDefaults;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestKafkaSecurityConfig
+{
+    @Test
+    public void testDefaults()
+    {
+        // No defaults, should be empty if nothing is set specifically
+        assertRecordedDefaults(recordDefaults(KafkaSecurityConfig.class)
+                .setKeystoreLocation(null)
+                .setKeystorePassword(null)
+                .setKeystoreType(null)
+                .setKeyPassword(null)
+                .setTruststoreLocation(null)
+                .setTruststorePassword(null)
+                .setTruststoreType(null)
+                .setEndpointIdentificationAlgorithm(null));
+    }
+
+    @Test
+    public void testExplicitPropertyMappings()
+            throws IOException
+    {
+        String secret = "confluent";
+        Path keystorePath = Files.createTempFile("keystore", ".jks");
+        Path truststorePath = Files.createTempFile("truststore", ".jks");
+
+        createTempFile(keystorePath, secret);
+        createTempFile(truststorePath, secret);
+
+        Map<String, String> properties = new ImmutableMap.Builder<String, String>()
+                .put("kafka.ssl.keystore.location", keystorePath.toString())
+                .put("kafka.ssl.keystore.password", "keystore-password")
+                .put("kafka.ssl.keystore.type", "JKS")
+                .put("kafka.ssl.key.password", "key-password")
+                .put("kafka.ssl.truststore.location", truststorePath.toString())
+                .put("kafka.ssl.truststore.password", "truststore-password")
+                .put("kafka.ssl.truststore.type", "JKS")
+                .put("kafka.ssl.endpoint-identification-algorithm", "disabled")
+                .build();
+        KafkaSecurityConfig expected = new KafkaSecurityConfig()
+                .setKeystoreLocation(keystorePath.toString())
+                .setKeystorePassword("keystore-password")
+                .setKeystoreType(KafkaKeystoreTruststoreType.JKS)
+                .setKeyPassword("key-password")
+                .setTruststoreLocation(truststorePath.toString())
+                .setTruststorePassword("truststore-password")
+                .setTruststoreType(KafkaKeystoreTruststoreType.JKS)
+                .setEndpointIdentificationAlgorithm(KafkaEndpointIdentificationAlgorithm.DISABLED);
+
+        assertFullMapping(properties, expected);
+    }
+
+    @Test
+    public void testEmptyPropertiesAreReturned()
+    {
+        KafkaSecurityConfig sut = new KafkaSecurityConfig();
+        assertThat(sut.getKafkaClientProperties().isEmpty());
```

Comment:
assertion is wrong. It should be:
```
        assertThat(sut.getKafkaClientProperties()).isEmpty();
```

That is the reason why people don't like fluent assertions. AKA false positive.